setup.cfg

```diff
@@ -4,7 +4,6 @@ logging-clear-handlers=1
 eval-attr="!=slow"
 
 [flake8]
-min-version = 2.7
```

Comment:
Is this going to cause us trouble when running flake8 outside of Travis or Tox? For example, my editor plugin ([syntastic](https://github.com/scrooloose/syntastic)) seems to pick up this configuration.